components/schema-migrator/migrations/director/20230505102050_create_operation.up.sql

```diff
@@ -0,0 +1,15 @@
+BEGIN;
```

Comment:
This [PR](https://github.com/kyma-incubator/compass/pull/3066) introduces a new migration. By the time we merge this change, the PR may have been already merged. This is a reminder to check the migration name before merging this change.